apps/meteor/client/views/admin/customEmoji/CustomEmojiRoute.tsx

```diff
@@ -45,7 +45,7 @@ const CustomEmojiRoute = (): ReactElement => {
 	return (
 		<Page flexDirection='row'>
 			<Page name='admin-emoji-custom'>
-				<Page.Header title={t('Custom_Emoji')}>
+				<Page.Header title={t('Emoji')}>
```

Comment:
There are other places that are still using `Custom_Emoji` token, should it be replaced by just `Emoji`, if so change it and remove the unused token... same to other tokens that it were replaced